manual/core/integration/README.md

```diff
@@ -0,0 +1,287 @@
+[INFO]
+[INFO] --- exec-maven-plugin:1.3.1:java (default-cli) @ yourapp ---
+11:39:45.355 [Main.main()] INFO  c.d.o.d.i.c.DefaultMavenCoordinates - DataStax Java driver for Apache Cassandra(R) (com.datastax.oss:java-driver-core) version 4.0.0-beta1
+11:39:45.648 [poc-admin-0] INFO  c.d.o.d.internal.core.time.Clock - Using native clock for microsecond precision
+11:39:45.649 [poc-admin-0] INFO  c.d.o.d.i.c.metadata.MetadataManager - [poc] No contact points provided, defaulting to /127.0.0.1:9042
+3.11.2
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 11.777 s
+[INFO] Finished at: 2018-06-18T11:32:49-08:00
+[INFO] Final Memory: 16M/277M
+[INFO] ------------------------------------------------------------------------
+```
+
+#### Gradle
+
+[Initialize a new project][gradle_init] with Gradle.
+
+Modify `build.gradle` to add the dependencies:
+
+```groovy
+group 'com.example.yourcompany'
+version '1.0.0-SNAPSHOT'
+
+apply plugin: 'java'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'com.datastax.oss', name: 'java-driver-core', version: '4.0.0-alpha4-SNAPSHOT'
+    compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
+}
+```
+
+Then place [application.conf](#application-conf), [logback.xml](#logback-xml) and
+[Main.java](#main-java) in the same locations, and with the same contents, as in the Maven example:
+
+```
+./src/main/resources/application.conf
+./src/main/resources/logback.xml
+./src/main/java/Main.java
+```
+
+Optionally, if you want to run from the command line, add the following at the end of
+`build.gradle`:
+
+```groovy
+task execute(type:JavaExec) {
+    main = 'Main'
+    classpath = sourceSets.main.runtimeClasspath
+}
+```
+
+Then launch with:
+
+```
+$ ./gradlew execute
+```
+
+You should see output similar to:
+
+```
+$ ./gradlew execute
+:compileJava
+:processResources
+:classes
+:execute
+13:32:25.339 [main] INFO  c.d.o.d.i.c.DefaultMavenCoordinates - DataStax Java driver for Apache Cassandra(R) (com.datastax.oss:java-driver-core) version 4.0.0-alpha4-SNAPSHOT
+13:32:25.682 [poc-admin-0] INFO  c.d.o.d.internal.core.time.Clock - Using native clock for microsecond precision
+13:32:25.683 [poc-admin-0] INFO  c.d.o.d.i.c.metadata.MetadataManager - [poc] No contact points provided, defaulting to /127.0.0.1:9042
+3.11.2
+
+BUILD SUCCESSFUL
+```
+
+#### Manually (from the binary tarball)
+
+If your build tool can't fetch dependencies from Maven central, we publish a binary tarball on the 
+[DataStax download server][downloads].
+
+The driver and its dependencies must be in the compile-time classpath. Application resources, such
+as `application.conf` and `logback.xml` in our previous examples, must be in the runtime classpath.
+
+### Component lifecycles
```

Comment:
I was going to put this under a "best practices" header, but I don't have a lot of other topics to cover. Any idea?

Reply:
We could probably expand this into a separate manual page for Best Practices, but I agree it makes sense to just leave this here as is for now.

Reply:
I ended up removing this section, everything is covered already in other parts of the manual.